examples/ads.amp.html

```diff
@@ -247,6 +248,30 @@ <h2 id="adgeneration">Ad Generation</h2>
     <div fallback></div>
   </amp-ad>
 
+  <h2 id="adhese">Adhese</h2>
+  <amp-ad width=300 height=250
+      type="adhese"
+      data-location="_sdk_amp_"
+      data-position=""
```

Comment:
Our request is always put together by these 5 components whereof the position can be empty yes.
I put it in the required components list because I would like for our customers to always use these 5 components as a minimum.